README.md

```diff
@@ -1,14 +1,58 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+     
+Axe.Windows is a [NuGet](https://www.nuget.org/) package for running automated accessibility tests on Windows® applications.
 
-# Contributing
+## Latest release
 
-This project welcomes contributions and suggestions.  Most contributions require you to agree to a
-Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
-the rights to use your contribution. For details, visit https://cla.microsoft.com.
+To get the latest version of the Axe.Windows NuGet package, visit
+[Axe.Windows on NuGet.org](https://www.nuget.org/packages/Axe.Windows/).
 
-When you submit a pull request, a CLA-bot will automatically determine whether you need to provide
-a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions
-provided by the bot. You will only need to do this once across all repos using our CLA.
+For details about the latest release, please see the [Release Notes Page](./docs/ReleaseNotes.md).
+     
+## Run Axe.Windows automated accessibility tests
+     
+For information about how to use Axe.Windows to test a Windows® application, please see the [Automation Guide](./docs/automation.md);
+     
+## Contributing
+All contributions are welcome! Please read through our guidelines on [contributions](./Contributing.md) to this project.
 
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
-For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
-contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+## Building the code
+You can find more information on how to set up your development environment [here](./docs/SetUpDevEnv.md).
+
+### 1. Clone the repository
+- Clone the repository using one of the following commands
+  ``` bash
+  git clone https://github.com/Microsoft/axe-windows.git
+  ```
+  or
+  ``` bash
+  git clone git@github.com:Microsoft/axe-windows.git
+  ```
+- Select the created directory
+  ``` bash
+  cd axe-windows
+  ```
+
+### 2. Open the solution in visual studio
+- Use the `src/AxeWindows.sln` file to open the solution.
+
+### 3. Build and run unit tests
+
+## Testing
+We use the unit test framework from Visual Studio. Find more information in our [FAQ section](./docs/FAQ.md).
+
+### More Information
```

Comment:
Seems unclear to me if we are using Sentence case or Title Case for our headings; in this file we seem to have both (although more of Sentence case).

This would be helpful in automation doc PR as well to know.

Reply:
The Microsoft style guide suggests we use sentence capitalization. Please see [here](https://docs.microsoft.com/en-us/style-guide/capitalization).